src/com/projectkorra/projectkorra/waterbending/WaterSpoutWave.java

```diff
@@ -252,7 +252,7 @@ public void progress() {
 				}
 			} else if (this.animation == AnimateState.TOWARD_PLAYER) {
 				this.revertBlocks();
-				final Location eyeLoc = this.player.getTargetBlock((HashSet<Material>) null, 2).getLocation();
+				final Location eyeLoc = this.player.getTargetBlock(null, 2).getLocation();
```

Comment:
Does this compile and run? I remember this being added due to java not being able to infer which method to use, since there was two that null as a parameter was applicable for.

Reply:
Tested on 1.16.5 server using latest version that build on source code.
![image](https://user-images.githubusercontent.com/44327892/194733424-9bbc4c59-60c8-456a-856f-b30211eeb88f.png)

				```
this.player.sendMessage("WaterSpoutWave.1");
				final Location eyeLoc = this.player.getTargetBlock(null, 2).getLocation();
				this.player.sendMessage("WaterSpoutWave.2");
```

It compiled and worked fine. 
1.8.8
![image](https://user-images.githubusercontent.com/44327892/194733577-eaa54d4a-6319-44c9-adba-45e68b02770a.png)

1.12.2
![image](https://user-images.githubusercontent.com/44327892/194733587-9e7a491d-b3bb-4e50-8e48-487022c38bf6.png)

1.16.5
![image](https://user-images.githubusercontent.com/44327892/194733579-4bd25df7-7faa-46fe-addf-2b619a939061.png)

Looks like that issue is resolved long time ago.